templates/index.html

```diff
@@ -6,514 +6,688 @@
+              <strong>Phil Eisler</strong>
+            </p>
+            <p class="u-text--muted">VP Cloud Streaming at NVIDIA</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/53e415bd-arm-logo.png",
+          alt="ARM logo",
+          width="852",
+          height="312",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-6">
+            <p>
+              <i>"It's no secret that TCO is a key driver of cloud provider decisions. And that's what gets Arm excited about the Android in the Cloud solution Ampere, Canonical and NETINT have developed. The Ampere Altra CPU packs 80 high-performance Neoverse N1 cores into a very power-efficient design that should deliver incredible TCO for Android in the Cloud use cases like cloud gaming, Android app development and IoT enablement to cloud providers."</i>
+            </p>
+          </div>
+          <div class="col-3">
+            <p class="u-no-margin--bottom">
+              <strong>Robbie Williamson</strong>
+            </p>
+            <p class="u-text--muted">Arm Senior Director of Solutions Engineering</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/bbb69a95-ampere-logo.png",
+          alt="Ampere logo",
+          width="852",
+          height="312",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-6">
+            <p>
+              <i>"The combination of Ampere's Arm-based servers with a provisioned virtualisation solution like Canonical's Anbox Cloud delivers the flexible, high-performance and secure infrastructure that developers need in order to deliver a better user experience for consumers."</i>
+            </p>
+          </div>
+          <div class="col-3">
+            <p class="u-no-margin--bottom">
+              <strong>Jeff Wittich</strong>
+            </p>
+            <p class="u-text--muted">SVP of Products at <a href="https://amperecomputing.com/">Ampere</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/8b62755e-equinix-logo-vw.png",
+          alt="Equinix logo",
+          width="80",
+          height="100",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
```

Comment:
@mattea-turic I couldn't find this image in the Figma file from the asset server. I used the same logo from the logo section above but resized it to make it smaller. Would this be okay?
<img width="140" alt="image" src="https://github.com/user-attachments/assets/3bb3afef-a01a-4153-a6ad-98910e99a41c" />